feat(loom): initial loom design

Mxrxk opened the pull request:
### Description
- Loom design for reference during fabrication.

### Checklist
- [x] N/A ~~ERC passes with no errors.~~
- [x] N/A ~~DRC passes with no errors.~~
- [x] No files which should not be under version control added (see [README](https://github.com/sufst/pcb/blob/main/README.md)).
- [x] Library paths are relative paths.

### Additional Notes
To be added:
- FSUK datalogger
- USB connection for umbilical
- Damper pots

Commit:
feat(Loom): initial loom design

t-bre commented on Loom/Accumulator.pdf:
Shouldn't be under version control

t-bre commented on Loom/Loom:
Shouldn't be under version control

t-bre commented on Loom/Loom.cir:
Shouldn't be under version control

t-bre commented on Loom/untitled.kicad_sch:
Untitled? Maybe needs to be deleted?

t-bre reviewed (changes requested):
Schematics are good, reviewed in detail in person yesterday.

Some stuff is in here under version control that shouldn't be (PDFs, exported net lists, etc). Best to remove before merging.

Probably best to just do `git rm <FILES>` and make a new commit deleting the stuff then it will be squashed away.

t-bre commented:
Maybe we want to tweak the .gitignore to prevent some of this stuff from being added accidentally in future. Just have to be careful of PDFs because they're allowed in the `release` folder.

Commit:
chore(Loom): removed items that shouldnt be in folder

t-bre commented on Loom/Dash Box.kicad_sch:
Sorry I missed this on the last one but small thing to change on the dash box for the LEDs. Currently for reference:

<img width="361" alt="Screenshot 2023-04-24 at 14 58 43" src="https://user-images.githubusercontent.com/31595761/234019024-d0c9ab08-ec04-4340-8810-906e900344b9.png">

What needs changed:
- Current limiting resistors need to be added to the `R2D`, `TS_ON` and `DRS` LEDs.
- I'd like all the current limiting resistors to be on the stripboard, external to the PCBs (even if they have footprints for them) so that it's easier to change the brightness just by modifying the stripboard rather than having to remove PCBs from the car and desolder stuff.
- Current limiting resistors should be on the -ve side of the LEDs.

t-bre reviewed (changes requested):
Sorry missed one thing in the last review, but all the previous stuff has been fixed 👍.

t-bre commented:
Also I've added `loom` to the projects table in the README (on the main branch), so no need to do that here.

Commit:
fix(Loom): made requested changes + chores

Mxrxk commented:
Changes:
- Changed "XLA+" to "HV+" and "XLA-" to "HV-"
- Altered the connectors for the Discharge and Hv_sense PCBs in the Junction Box
- Altered how the cables leave the Junction box
- Altered how the cables leave the Pedal box
- Fixed library path related issue
- Renamed some Netlists (wires)
- Added current limiting resistors to Dash PCB

Commit:
fix(Loom): changed position of components

Mxrxk commented:
Changes:
- Changed side on which the current limiting resistors inside the Dash box are on. Moved from positive side to negative side
- Notes indicating more information about the resistors being on strip boards
- Notes indicating more information about box "Nozzles"

t-bre committed:
feat(loom): minor tweaks to dash + VCU connections

t-bre committed:
chore(loom): rename folder for consistency

t-bre reviewed (approved):
Made a couple minor tweaks (added VC LEDs connection net label) but looks good!
